### DISCUSSION TITLE
Block third party fonts

### DISCUSSION THREAD

**Discussion by Universalizer** [uazo/bromite-buildtools#102] [Ideas]
First of all, Thank You for considering my request, for opening these 'discussions section' in your repo,👍

👀 In the past, had seen :-
https://github.com/uazo/bromite-buildtools/issues/52
https://github.com/uazo/bromite-buildtools/issues/60

### Check `or read and note it`, these filter following :- 
https://raw.githubusercontent.com/yokoffing/filterlists/main/block_third_party_fonts.txt

Case 1 :-
Probably **to block third party fonts `ONLY`,** (downloading from server-side / sites, to user's browser),

Case 2 :-
But, fingerprinting test ⁉️

👎 Generally, 'testing sites' demands `to turn on javascript necessarily,`

⁉️❓ Cookies optionally.

Hoping, these highlighting discussion may be considering.

**Comment by uazo** [comment]
I have difficulty understanding... but I try to answer

> downloading from server-side

downloading fonts from the web is not really a problem, downloading fonts is like downloading images (or javascript, or any other resource).

> 'testing sites' demands to turn on javascript necessarily

is not entirely true. it is also possible to simply use css.

> Cookies optionally.

this one is really cryptic!

**Reply by Universalizer** [reply]
> is not entirely true. it is also possible to simply use css.

Yes, css also.

**Comment by Universalizer** [comment]
https://github.com/Universalizer/Universal-FilterLists#fonts
> Fonts
Block third party fonts | [add](https://raw.githubusercontent.com/yokoffing/filterlists/main/block_third_party_fonts.txt)
[optional] A successor to Fanboy's Anti-thirdparty Fonts, this option blocks all web fonts from third-party providers but allows the browser to download fonts from first-party domains. I prefer this method as it does not require the extra connection to a third-party domain (speed and privacy); and, since I trust the first-party site enough to connect to it, I do not take on much additional privacy or security risks when I allow my browser to download a font directly from them.[1](https://collinmbarrett.com/block-web-fonts/) Note: This will break the "look and feel" of many sites.

**Comment by Universalizer** [comment]
https://www.dailymail.co.uk/sciencetech/article-11970491/Google-using-FONTS-track-online-it.html
18:22 BST 22 Apr 2023 , updated 18:22 BST 22 Apr 2023
